addons/html_editor/static/src/main/table/table_plugin.js

```diff
@@ -761,6 +761,7 @@ export class TablePlugin extends Plugin {
                     table.classList.toggle("o_selected_table", true);
                     for (const td of getTableCells(table)) {
                         td.classList.toggle("o_selected_td", true);
+                        this.dispatchTo("deselect_custom_selected_nodes_handlers", td);
```

Comment:
I don't understand the need for these two calls to `deselect_custom_selected_nodes_handlers`. I would assume that those two functions that select table cells are called as a result of some selectionchange so, if the selection is not around the HR anymore then it should have been cleaned by the selectionchange handler from the separator plugin, shouldn't it? If it doesn't work, how comes the selection change handler of the separator plugin is not called?

Reply:
It handles case where a separator is inside a table that is selected. Since the HR element is part of the table, it is also included in the selection. As a result, both the separator and the table will have a custom selection applied. Therefore, `deselect_custom_selected_nodes_handlers` remove the `hr` elements custom selection when the table's custom selection is applied.